### PR TITLE
Add Ryan Jones as non-code maintainer

### PR DESCRIPTION
**Pull request by lucassaldanha**
## Proposal

I propose to add @ryanjjones10 as a Besu non-code maintainer.

@ryanjjones10 has contributed with roadmap planning and categorizing bugs and issues for Besu. Becoming a non-code maintainer, Ryan will be able to label bugs and close relevant tickets, helping to keep the project tidy.

Voting ends two weeks from today.

For more information on this process see the Becoming a Maintainer section in the MAINTAINERS.md file.

## Other changes

I have also taken this opportunity to fix the ordering in the maintainers list. It is supposed to be alphabetical on the Github handle, not the name.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -8,31 +8,32 @@
...
 | Mark Terry       | mark-terry       | m.terry          |
 | Karim Taam       | matkt            | matkt            |
 | Meredith Baxter  | mbaxter          | mbaxter          |
 | Nicolas Massart  | NicolasMassart   | NicolasMassart   |
-| Sajida Zouarhi   | sajz             | SajidaZ          |
-| Simon Dudley     | siladu           | siladu           |
 | Stefan Pingel    | pinges           | pinges           |
-| Taccat Isid      | taccatisid       | taccatisid       |
 | Rai Sur          | RatanRSur        | ratanraisur      |
+| Ryan Jones       | ryanjjones10     | ryanjjones10     |
```
Comment (lucassaldanha) [inline MAINTAINERS.md:32]: This is the only new maintainer added in this PR. Other changes are just shuffling names around.

File: MAINTAINERS.md [hunk 2]
```diff
@@ -8,31 +8,32 @@
...
+| Sajida Zouarhi   | sajz             | SajidaZ          |
+| Simon Dudley     | siladu           | siladu           |
```
Comment (macfarla) [inline MAINTAINERS.md:34]: i comes after h @lucassaldanha

**Review by frankisawesome — APPROVED** [review]

**Review by garyschulte — APPROVED** [review]

**Review by jframe — APPROVED** [review]

**Review by matkt — APPROVED** [review]

**Review by daniellehrner — APPROVED** [review]

**Review by fab-10 — APPROVED** [review]

**Review by ajsutton — APPROVED** [review]

**Review by macfarla — APPROVED** [review]

**Review by shemnon — CHANGES_REQUESTED** [review]
One concern I have is the visibility of Ryan's work.  There are no links to any of the activities that Ryan has contributed. Other maintainers have had a list of notable contributions to their maintainer proposal.

The lack of public visibility raises several issues. As it stands without publicly linkable activity one would not be counted as a contributor for TSC elections, per the hyperledger charter. 
Second, the role of maintainer should be a public one, not a role that provides services in a private setting. It also comes with duties that are impossible to fulfill without a public presence.

If the issue is needing to get github write access for zenhub, that can be granted without elevating someone to the role of maintainer. The mechanics of github should not be conflated with the definition of maintainer in the Hyperledger charter.

**Comment by ryanjjones10** [comment]
@shemnon, I certainly agree with you. What is the best way to get the github write access for Zenhub?

Over time, I look to contribute more to Besu in a public setting and will look for ways to do so.

**Comment by ryjones** [comment]
> @shemnon, I certainly agree with you. What is the best way to get the github write access for Zenhub?

please send email to community-architects@hyperledger.org

**Review by jflo — CHANGES_REQUESTED** [review]
I agree with Danno. Ryan is a swell guy, and will make great and needed contributions in the near future. However, right now the Maintainer status carries with it responsibilities that aren't really fit for non-committers. Things like reviewing PRs and executing releases.

Ryan, just know that we will find a solution to this to get you involved, and so you have the access you need to organize outstanding issues.

The re-organization of names on the list is fine, that can stay.

**Comment by lucassaldanha** [comment]
Thanks for the feedback. I'll close this PR and we will try to get the permissions sorted instead of adding Ryan to the maintainers list.

**Comment by macfarla** [comment]
@lucassaldanha if you've still got it handy can you push a PR with just the reordering on github ID?

**Comment by lucassaldanha** [comment]
> @lucassaldanha if you've still got it handy can you push a PR with just the reordering on github ID?
Sure!

**Pull request closed without merging**